core/services/fluxmonitor/flux_monitor.go

```diff
@@ -411,8 +438,10 @@ func NewPollingDeviationChecker(
 }
 
 const (
-	PriorityNewRoundLog      uint = 0
-	PriorityAnswerUpdatedLog uint = 1
+	PriorityFlagRaisedLog    uint = 0
```

Comment:
I think we're actually going to want a single priority for both raised and lowered flags, or else they might end up being received out of order.  For example, if a "lowered" flag arrives, and then a "raised" flag, FM will process "lowered" last, causing it to hibernate when it shouldn't.  Assuming the hibernation logic you've written is idempotent, we should be able to keep the queue for the "flags" priority at a length of 1.  If it's not, we should keep 2.